src/Native/Windows/clrcompression/CMakeLists.txt

```diff
@@ -77,8 +77,10 @@ else()
     )
 endif()
 
-if ($ENV{__VSVersion} STREQUAL vs2015)
-    add_definitions(-DHAVE_VSNPRINTF)
+include (CheckSymbolExists)
```

Comment:
Do we need this `HAVE_VSNPRINTF` at all? It was done for VS2015 only in the original.

Reply:
Right because at that time VS2017 didn't even exist. I looked at the history and that particular code wasn't touched after it was open sourced. Unfortunately I don't have much context here.

Reply:
Removal would be better. Internally zlib[-intel] has another macro `#define NO_VSNPRINTF` in case old compilers and implementation  (`.c`) files only uses `NO_VSNPRINTF`.

Reply:
Removal would be better. Internally zlib[-intel] has another macro `#define NO_VSNPRINTF` in case old compilers and implementation  (`.c`) files only use `NO_VSNPRINTF`.

Reply:
Yes, builds fine without it.